### DISCUSSION TITLE
How to show scrollbar in the main window?

### DISCUSSION THREAD

**Discussion by X-m7** [tauri-apps/tauri#5501] [Q&A] [answered]
Currently when my app displays content that ends up below the window area there is no scrollbar that I can see. Is it meant to be disabled by default, and if so is there a way to enable it anywhere? Setting the `overflow-y: scroll;` CSS property on the body does not do the trick, although scrolling does actually work, there is just no visual indication that it is possible. Setting `overflow-y: hidden;` does disable scrolling, so I assume the CSS property did actually get applied.

I am using Tauri 1.1.1, running on Arch Linux with a KDE Plasma 5.26.2 Wayland session if that matters.

Below is a screenshot of my issue, where my mouse cursor is hovering on the right edge of the window:
![image](https://user-images.githubusercontent.com/18729369/198559528-f0179bca-b9e8-4dd6-b895-415f10f61b96.png)

**Comment by pypp** [comment]
Can you share more of your code ?

**Comment by X-m7** [comment, marked as answer]
Just found the cause of the problem, the GTK theme I was using (Breeze from KDE Plasma) appears to break the scrollbar in Tauri, switching back to Adwaita makes the scrollbar appear again.

**Reply by wvffle** [reply]
Well, forcing user to switch the theme is not a solution. The problem is either with the Breeze theme or Tauri itself.
